packages/mobile/src/components/now-playing-drawer/ActionsBar.tsx

```diff
@@ -118,6 +119,11 @@ export const ActionsBar = ({ track }: ActionsBarProps) => {
   const { onOpen: openPremiumContentPurchaseModal } =
     usePremiumContentPurchaseModal()
 
+  const { data: albumInfo } = trpc.tracks.getAlbumBacklink.useQuery(
```

Comment:
Hmm I with about a new call every track tile. I fired this groups calls in trpc land?

Reply:
One business requirement also is that if a user is on an album and has the list of tracks we probably don't want to back link those?

Reply:
Yes, good call. I think I will merge as is but follow up with that.

Reply:
Re: calling the same fetch on each instance, the trpc hook will cache by the args (trackId) and batch requests so it should stay pretty efficient even when requesting for many tracks in a lineup.